bitshuffle/bitshuffle.py

```diff
@@ -188,6 +190,10 @@ def main():
                              "Ignored if decoding packets. " +
                              "Currently supported: 'bz2', 'gzip'.")
 
+    parser.add_argument("--message", "-s", default=default_msg,
```

Comment:
I like this option, but the short flag is non intuitive. Perhaps we could change this to '-m' and compresslevel (which currently uses '-m') to '-l'.